autopush-common/src/reliability.rs

```diff
@@ -211,25 +225,64 @@ impl PushReliability {
                 .unwrap_or_else(|| "None".to_owned()),
             new
         );
+
+        let state_key = format!("state.{id}");
+        trace!("🔍 state key: {}", &state_key);
+        // The first state is special, since we need to create the `state_key`.
+        if new == ReliabilityState::Received {
+            trace!("🔍 Creating new record");
+            // we can't perform this in a transaction because we can only increment if the set succeeds.
+            // Create the new `state.{id}` key if it does not exist, and set the expiration.
+            let options = redis::SetOptions::default()
+                .with_expiration(redis::SetExpiry::EX(expr.unwrap_or(0)))
```

Comment:
Expiry to `0`, does it mean it expires right away? Should this be a `const` or `enum` instead of this magic number?